docs/guides/auto-generating-selectors.md

```diff
@@ -65,6 +65,54 @@ const bears = useBearStore.use.bears()
 const increment = useBearStore.use.increment()
 ```
 
+## Vanilla Store
+
+If you are using a vanilla store, then consider the function below.
+
+```typescript
+import { useStore, StoreApi } from "zustand";
+
+type WithSelectors<S> = S extends { getState: () => infer T }
+  ? S & { use: { [K in keyof T]: () => T[K] } }
+  : never;
+const createSelectors = <S extends StoreApi<object>>(_store: S) => {
+  const store = _store as WithSelectors<typeof _store>;
+  store.use = {};
+  for (const k of Object.keys(store.getState())) {
+    (store.use as any)[k] = () =>
+      useStore(_store, (s) => s[k as keyof typeof s]);
+  }
+
+  return store;
+};
```

Comment:
The code style is inconsistent with Zustand's one. Please adjust according to the code snippets above.

Reply:
Hi @sewera! I did some refactoring trying to match Zustand code style based on the above snippets. Can you check again?